internal/suites/example/compose/traefik/docker-compose.yml

```diff
@@ -1,7 +1,8 @@
+---
 version: '3'
 services:
   traefik:
-    image: traefik:v1.7.29-alpine
+    image: traefik:1.7.28-alpine
```

Comment:
Are we sure we want to downgrade?

Reply:
It was upgraded in master since forking I believe. Will be fixed once I merge master into this.